docs/how-to-use-relational-backend-storage.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to use relational backend storage
+slug: /how-to-use-relational-backend-storage
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Introduction
+
+Before the version `0.5.0`, Gravitino only supports KV backend storage to store metadata. Since
+RDBMS is widely used in the industry, starting from the version `0.5.0`, Gravitino supports using
+RDBMS as relational backend storage to store metadata. This doc will guide you on how to use the
+relational backend storage in Gravitino.
+
+Relational backend storage mainly aims to the users who are accustomed to using `RDBMS` to
+store data or lack available a KV storage, and want to use Gravitino.
+
+With relational backend storage, you can quickly deploy Gravitino in a production environment and
+take advantage of relational storage to manage metadata.
+
+### What kind of backend storage are supported
+
+Currently, relational backend storage supports the `JDBCBackend`, and uses `MySQL` as the
+default storage for `JDBC Backend`.
```

Comment:
JDBC Backend -> JDBCBackend?

Reply:
JDBCBackend is a specific configuration when you quote with "``", so if you want to claim as normal words, then it can be used as JDBC backend without quote, otherwise, you'd better do with `JDBCBackend`.